<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Error Validations(failed)" verbose="0">
    <classes>
      <class name="datadriven.AutomationFrameworkTesting.ErrorValidations">
        <methods>
          <include name="launchApplication"/>
          <include name="submitOrder1"/>
        </methods>
      </class> <!-- datadriven.AutomationFrameworkTesting.ErrorValidations -->
    </classes>
  </test> <!-- Error Validations(failed) -->
  <test thread-count="5" parallel="tests" name="Submit Order(failed)" verbose="0">
    <classes>
      <class name="datadriven.AutomationFrameworkTesting.StandAloneTest">
        <methods>
          <include name="launchApplication"/>
          <include name="orderHistory"/>
          <include name="submitOrder"/>
        </methods>
      </class> <!-- datadriven.AutomationFrameworkTesting.StandAloneTest -->
    </classes>
  </test> <!-- Submit Order(failed) -->
</suite> <!-- Failed suite [Suite] -->
